srtcore/buffer_tools.h

```diff
@@ -188,14 +187,53 @@ class CSndRateEstimator
         bool empty() const { return m_iPktsCount == 0; }
     };
 
-    int incSampleIdx(int val, int inc = 1) const;
+private:
+    static const int NUM_PERIODS        = 10;
+    static const int SAMPLE_DURATION_MS = 100; // 100 ms
 
     Sample m_Samples[NUM_PERIODS];
 
     time_point m_tsFirstSampleTime; //< Start time of the first sample.
     int        m_iFirstSampleIdx;   //< Index of the first sample.
     int        m_iCurSampleIdx;     //< Index of the current sample being collected.
     int        m_iRateBps;          //< Rate in Bytes/sec.
+
+    int incSampleIdx(int val, int inc = 1) const;
+};
+
+class CMobileRateEstimator:CSndRateEstimator
+{
+    typedef sync::steady_clock::time_point time_point;
+
+public:
+    CMobileRateEstimator();
+
+    /// Add sample.
+    /// @param [in] pkts   number of packets in the sample.
+    /// @param [in] bytes  number of payload bytes in the sample.
+    void addSample(int pkts = 0, double bytes = 0);
+
+    /// Clean the mobile measures table to reset average value.
+    void resetMeasuresTable() {resetMeasuresTable(0,NUM_PERIODS);};
```

Comment:
```suggestion
    void resetMeasuresTable() { resetMeasuresTable(0,NUM_PERIODS); };
```

Reply:
Done.

Reply:
You likely forgot to push your changes to GitHub.